### DISCUSSION TITLE
Temporary Folder Environment Error

### DISCUSSION THREAD

**Discussion by BYTEMEE1** [undergroundwires/privacy.sexy#145] [General]
I believe "Privacy.sexy" removal of the %USERPROFILE%\AppData\Local\Microsoft\Windows\Temporary Internet Files and/or INetCache folder in some configurations causes Microsoft Office apps like Word and Excel not to launch properly and produces the error message: "Word could not create the work file. Check the Temp environment variable“. If it happens to you, [this site](https://thegeekpage.com/word-could-not-create-the-work-file/#Method_5_Delete_the_Office_64-bit_Previewer_Key_from_Registry_Editor) was a big help in fixing the issue. I resolved my problem by deleting the registry key containing: "{84F66100-FF7C-4fb4-B0C0-02CD7FB668FE}"

Maybe that will help someone.

**Comment by undergroundwires** [comment]
Hi, I just changed the way directories are being deleted/cleaned up in privacy.sexy with 060e7896624309aebd25e8b190c127282de177e8. It now retains `INetCache` and `Temporary Internet Files` directories, purging only their contents. Hopefully this resolves the issue, I recommend creating bug reports for quicker actions.
